_includes/pbc/all/install-features/202204.0/install-the-product-configuration-feature.md

```diff
@@ -2133,46 +2100,10 @@ class ApplicationDependencyProvider extends SprykerApplicationDependencyProvider
 }
 ```
 
-{% info_block warningBox "Verification" %}
-
-Make sure that the Twig configuration method `getZedDirectoryPathPatterns` has been expanded with additional paths for `ExampleDateTimeProductConfigurationRenderStrategyPlugin`.
-
-{% endinfo_block %}
-
-**src/Pyz/Zed/Twig/TwigConfig.php**
-
-```php
-<?php
-
-<?php
-
-namespace Pyz\Zed\Twig;
-
-use Spryker\Zed\Twig\TwigConfig as SprykerTwigConfig;
-
-class TwigConfig extends SprykerTwigConfig
-{
-  public function getZedDirectoryPathPatterns(): array
-  {
-      $directories = array_merge(
-          glob('vendor/spryker/spryker/Bundles/*/src/*/Zed/*/Presentation', GLOB_NOSORT | GLOB_ONLYDIR),
-          glob('vendor/spryker/spryker-shop/Bundles/*/src/*/Zed/*/Presentation', GLOB_NOSORT | GLOB_ONLYDIR)
-      );
-
-      $directories = array_merge(
-          $directories,
-          parent::getZedDirectoryPathPatterns()
-      );
-      sort($directories);
-      return $directories;
-  }
-}
```

Comment:
same here

Reply:
Yes, but it's IG for projects, as I see comment on that file:
`Only needed in Project, not in demoshop` - we should keep it.

Reply:
No, these paths related only to `spryker/spryker` and `spryker/spryker-shop` repositories. Normally projects will have `'vendor/spryker/*/src/*/Zed/*/Presentation'` and `'vendor/spryker-shop/*/src/*/Zed/*/Presentation'` paths and they are covered by config on a core level.